processing/data_collection/gazette/spiders/rj_petropolis.py

```diff
@@ -0,0 +1,61 @@
+import os
+import re
+from datetime import datetime
+from urllib.parse import urljoin
+
+import scrapy
+import dateparser
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjPetropolis(BaseGazetteSpider):
+    TERRITORY_ID = "3303906"
+    BASE_URL = "http://www.petropolis.rj.gov.br"
+
+    name = "rj_petropolis"
+    allowed_domains = ["petropolis.rj.gov.br"]
+    start_urls = [
+        "http://www.petropolis.rj.gov.br/pmp/index.php/servicos-na-web/informacoes/diario-oficial/viewcategory/3-diario-oficial.html"
+    ]
+    set_to_discard = {
```

Comment:
this seems like a hack because the implementation is not perfect or because the selectors are too broad.